include/browsers/chromium/Chromium.hh

```diff
@@ -0,0 +1,85 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2025 Michael Jonker
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/*****************************************************************************
+ * This file contains interfaces and methods for working with the CEF *
+ * embedded browser.                                                         *
+ *                                                                           *
+ *      *
+ *                                                                           *
+ *****************************************************************************/
+
+#ifndef GBW_BROWSERS_CHROMIUM_CHROMIUM_HH
+#define GBW_BROWSERS_CHROMIUM_CHROMIUM_HH // Why the doubling of the name - Q for Jonker?
```

Comment:
This is the C++ standards compliant way of doing `#pragma ONCE`, which has problems under some compiler contexts.

It is a guard that ensures that the compiler only includes the header once, so the definition MUST be a unique string. The naming convention is all UPPER_CASE: `GBW_` + \<file path and name\>, replacing `/` with `_`.

"Chromium.hh" happens to be under the "chromium" directory, thus `CHROMIUM_CHROMIUM`